linux_os/guide/system/permissions/partitions/mount_option_nodev_nonroot_local_partitions/bash/rhel6.sh

```diff
@@ -65,10 +63,8 @@ do
             # Replace old mount options for particular /etc/fstab's row (for this target
             # and fstype) with new mount options
             sed -i "s#${TARGET_HEAD}\(.*\)${TARGET_TAIL}#${TARGET_HEAD}${MOUNT_OPTIONS}${TARGET_TAIL}#" /etc/fstab
-
+            # Reset IFS back to default
+            IFS="$IFS_BKP"
```

Comment:
Why reset `IFS` every iteration?

Reply:
Basically this minimizes the time when `IFS` is different from what is the common expectation.